src/main/java/ro/societateahermes/backendservice/entities/DTO/CDMemberDTO.java

```diff
@@ -0,0 +1,33 @@
+package ro.societateahermes.backendservice.entities.DTO;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Pattern;
+
+@Entity
+@Data
```

Comment:
Dto-ul nu este ceva ce va fi stocat in baza de data, asa ca sterge adnotarile @Entity, @Id si @GeneratedValue

Reply:
done :)